lmdeploy/cli/serve.py

```diff
@@ -216,7 +234,12 @@ def add_parser_proxy():
         parser.set_defaults(run=SubCliServe.proxy)
         parser.add_argument('--server-name', type=str, default='0.0.0.0', help='Host ip for proxy serving')
         parser.add_argument('--server-port', type=int, default=8000, help='Server port of the proxy')
-        parser.add_argument('--strategy',
+        parser.add_argument('--serving-strategy',
+                            type=str,
+                            choices=['Hybrid', 'DistServe'],
+                            default='Hybrid',
+                            help='the strategy to dispatch requests to nodes')
```

Comment:
May clarify the help info. It is the same as `--routing-strategy`.

Reply:
The api_server is assigned a specific "role" in this PR. I propose updating the communication protocol between the api_server and the proxy_server to include this role information.

**Benefits:**

- The proxy server can make decisions based on the api_server's role.
- This change would allow us to remove the `--serving-strategy` option, simplifying the argument list.

Would this approach be feasible? I’d appreciate any feedback or suggestions.

Reply:
Agree. `--serving-strategy` can be removed and pd mode can be inferred from `engine role` in backend config.